torch/_tensor_docs.py

```diff
@@ -1903,15 +1903,27 @@ def callable(a, b) -> number
 q_per_channel_scales() -> Tensor
 
 Given a Tensor quantized by linear (affine) per-channel quantization,
-returns a Tensor of scales of the underlying quantizer().
+returns a Tensor of scales of the underlying quantizer. It has the number of
+elements that matches the corresponding dimensions (from q_per_channel_axis) of
+the tensor.
 """)
 
 add_docstr_all('q_per_channel_zero_points',
                r"""
 q_per_channel_zero_points() -> Tensor
 
 Given a Tensor quantized by linear (affine) per-channel quantization,
-returns a tensor of zero_points of the underlying quantizer().
+returns a tensor of zero_points of the underlying quantizer. It has the number of
+elements that matches the corresponding dimensions (from q_per_channel_axis) of
+the tensor.
+""")
+
+add_docstr_all('q_per_channel_axis',
+               r"""
+q_per_channel_zero_points() -> tuple of ints
+
+Given a Tensor quantized by linear (affine) per-channel quantization,
+returns a indices of dimensions on which per-channel quantization is applied.
```

Comment:
Do we really support a tuple for axis? Why not restrict axis to be scalar/list of length 1 and clarify this in the API.
Also NIT: the doc string  should say q_per_channel_axis()

Reply:
How likely are we to have multidim in the future? Having both apis would be confusing